scripts/docgen/docgen.php

```diff
@@ -1085,7 +1116,7 @@ function get_default_role ($role, $funcname, $default) {
 	switch ($opt) {
 		case 'v':
 		case 'version':
-			printf("%s\n", '$Revision$');
+			printf("%s\n", '$Revision: 336015 $');
```

Comment:
Unwanted change. Please leave it as it was.

Reply:
Thanks, done.